assets/grid-collapsed.css

```diff
@@ -0,0 +1,51 @@
+/* 
+  --collapsed-offset-x/y: the distance to offset an item's border onto the siblings below and to the right
+  --collapsed-shadow-offset-x/y: the distance to offset the overlapping shadows caused by offsetting grid items
```

Comment:
When setting the opacity of the shadow pretty high, we can see a gap between each item. It's nitpick, not sure if that's ok or not: 
![](https://screenshot.click/15-58-ftj45-4arw2.png)

Reply:
There's not much I can do about that when blur radius is applied. I've just ensured the shadows don't unnecessarily overlap due to the offset trickery. These shadows are still coming from individual elements so that's just how css shadows will behave. 

The only real solution to that is to use `filter: drop-shadow()` on the whole list rather than each item. The problem we'll run into with that is the focus outlines will be reflected in the shadow as well. Unless we apply a new focus outline design for cards that stays within the physical boundaries of the card.

Reply:
@YoannJailin @melissaperreault worth running this by you. 
Here's a video that shows the effect a bit more https://screenshot.click/16-40-glmnf-aa6pc.mp4

Here's a video demonstrate the alternative shadow approach https://screenshot.click/16-45-0dnob-t00et.mp4
But even if we're ok with that, the standard card case [ruins everyone's fun](https://screenshot.click/16-49-h19fv-8jy1w.png)

Reply:
Added this as a known issue in the PR description.